nixos/roles/gitlab.nix

```diff
@@ -267,6 +267,8 @@ in
 
     # Needed for Git via SSH.
     users.users.gitlab.extraGroups = [ "login" ];
+
+    systemd.services.gitlab-mailroom.serviceConfig.Restart = lib.mkForce "always";
```

Comment:
`lib.mkOverride 20` would be a better option for such cases to allow others to use mkForce, but it's acceptable here to use mkForce.